pa_matchbatch.sh

```diff
@@ -2,9 +2,21 @@
 set -e
 unset CDPATH
 
+# Arguments:
+# - REPO: Which repo is being released. `puppet-agent` or `puppet-agent-private`
+#         are the two primary options.
+# - WORKSPACE: Where the dependencies should be downloaded, e.g. `temp`.
+# - IGNORE_FOR: Which repos should be ignored (space-separated).
+# - OVERRIDE_PATH: A path to a txt file that will override versions.
+#
+# Deprecated arguments:
+# - PUPPET_AGENT_URL: This will still be read, but `REPO` is preferred.
+
 # where to find the agent
-puppet_agent_repo="git@github.com:puppetlabs/puppet-agent.git"
+REPO=${REPO:-puppet-agent}
```

Comment:
Should probably update the top-level `README` with this argument.

Reply:
Added.